src/js/components/Box.js

```diff
@@ -153,6 +159,7 @@ Box.propTypes = {
   alignContent: PropTypes.oneOf(['start', 'center', 'end', 'between', 'around', 'stretch']),
   appCentered: PropTypes.bool,
   backgroundImage: PropTypes.string,
+  children: PropTypes.any,
```

Comment:
Why do we need to add "children" as a formal propType? I had always thought that children was a special property that didn't need to be declared.